.circleci/config.yml

```diff
@@ -218,6 +253,14 @@ workflows:
             branches:
               only:
                 - develop
+      - screenshot_api_16:
+          requires:
+            - test_api_version_16
+            - test_api_version_25
```

Comment:
- `test_api_v16`
- `test_api_v25`

Reply:
Yes this is the workflows:

- Test API 16 ---\
                            ----- Screenshot API 16
- Test API 25 ---\

Later we can take screenshot for table or other devices

Reply:
Yes, but the idea is to short the names of the labels from `test_api_version_16` to `test_api_v16` for a better view of the graph in the workflow.